rdt-open-reader/src/main/assets/step-config/step-state-config.json

```diff
@@ -19,6 +19,5 @@
   "scan_qr_page": "step5",
   "scan_carestart_page": "step4",
   "twenty_min_countdown_timer_page": "step13",
-  "take_image_of_rdt_page": "step15",
-  "unique_batch_id_page": "step5"
```

Comment:
Any reason this is removed?

Reply:
just `unique_batch_id_page` has been moved to `covidindonesia/.../step-state-config.json` because it is being used just in covidindonesia flavor and its absence causes failure in filling Batch Id on last page.